.github/workflows/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM centos as centos-python
 RUN yum install -y make gcc openssl-devel libffi-devel sqlite-devel
 ARG PYTHON_VER
-ENV LC_CTYPE="en_US.UTF-8"
+ENV LC_ALL C.UTF-8
```

Comment:
@hbrunn I never really understand what I am doing when I touch those LC_ variables. Would you mind explaining why this is better or could you point me to a good explanation ?

Reply:
I find https://www.gnu.org/software/libc/manual/html_node/Locale-Categories.html quite illuminating

Reply:
@hbrunn Thanks !